lib/autoprefixer.js

```diff
@@ -30,13 +37,14 @@ module.exports = function () {
         // Prevent If-None-Match revalidation with the downstream middleware with ETags that aren't suffixed with "-autoprefixer":
         var ifNoneMatch = req.headers['if-none-match'];
         if (ifNoneMatch) {
+            var autoprefixerEtagRegExp = new RegExp(`-autoprefixer\\[${autoprefixerHash}\\]["-]$`);
             var validIfNoneMatchTokens = ifNoneMatch.split(" ").filter(function (etag) {
-                return (/-autoprefixer["-]$/).test(etag);
+                return autoprefixerEtagRegExp.test(etag);
```

Comment:
We might be able to get by with:

```js
const validIfNoneMatchTokens = ifNoneMatch.split(" ")
    .filter(etag => etag.includes(`-autoprefixer[${autoprefixerHash}]`));
```

It's probably not that important to assert that the next character is either a doublequote or a dash.